x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations_settings/curations_settings.tsx

```diff
@@ -50,26 +51,17 @@ export const CurationsSettings: React.FC = () => {
     toggleCurationsMode,
   } = useActions(CurationsSettingsLogic);
 
-  const { isLogRetentionUpdating, logRetention } = useValues(LogRetentionLogic);
-  const { fetchLogRetention } = useActions(LogRetentionLogic);
-
-  const analyticsDisabled = !logRetention?.[LogRetentionOptions.Analytics].enabled;
-
-  useEffect(() => {
-    if (hasPlatinumLicense) {
-      fetchLogRetention();
-    }
-  }, [hasPlatinumLicense]);
+  const {
+    engine: { analytics_enabled: analyticsEnabled },
+  } = useValues(EngineLogic);
 
   useEffect(() => {
-    if (logRetention) {
-      if (!analyticsDisabled) {
-        loadCurationsSettings();
-      } else {
-        onSkipLoadingCurationsSettings();
-      }
+    if (analyticsEnabled && dataLoading) {
+      loadCurationsSettings();
+    } else {
+      onSkipLoadingCurationsSettings();
     }
-  }, [logRetention]);
+  });
```

Comment:
```suggestion
  }, [analyticsEnabled, dataLoading]);
```

I think you'll need this, otherwise the useEffect will be triggered every render.